tests/lib/util/fixture-parser.js

```diff
@@ -0,0 +1,16 @@
+"use strict";
```

Comment:
I think `tests/lib/util` is intended to contain tests for the files in `lib/util` folder, not utilities for testing. However, I'm not sure of the correct place to put this file -- maybe `tests/fixtures`?